src/features/swap/index.tsx

```diff
@@ -0,0 +1,207 @@
+            // @ts-ignore
+            case 'ORDER_FULFILLED':
+              dispatch(
+                showNotification({
+                  title: 'Order fullfilled',
+                  message: 'Order was fullfilled',
+                  groupKey,
+                  variant: 'info',
+                }),
+              )
+              break
+            // @ts-ignore
+            case 'ORDER_EXPIRED':
+              dispatch(
+                showNotification({
+                  title: 'Order expired',
+                  message,
+                  groupKey,
+                  variant: 'warning',
+                }),
+              )
+              break
+            // @ts-ignore
+            case 'ORDER_CANCELLED':
+              dispatch(
+                showNotification({
+                  title: 'Order cancelled',
+                  message,
+                  groupKey,
+                  variant: 'warning',
+                }),
+              )
+              break
+          }
+        },
+      },
+    ]
+  }, [dispatch])
+
+  const [params, setParams] = useState<CowSwapWidgetParams | null>(null)
+  useEffect(() => {
+    setParams({
+      appCode: 'Safe Wallet Swaps', // Name of your app (max 50 characters)
+      width: '100%', // Width in pixels (or 100% to use all available space)
+      height: '860px',
+      // provider: safeAppWeb3Provider, // Ethereum EIP-1193 provider. For a quick test, you can pass `window.ethereum`, but consider using something like https://web3modal.com
+      chainId, // 1 (Mainnet), 5 (Goerli), 100 (Gnosis)
+      // standaloneMode: false,
+      disableToastMessages: true,
+      disablePostedOrderConfirmationModal: true,
+      hideLogo: true,
+      hideNetworkSelector: true,
+      sounds: {
+        orderError: null,
+        orderExecuted: null,
+        postOrder: null,
+      },
+      // tokenLists: [
+      //   // All default enabled token lists. Also see https://tokenlists.org
+      //   'https://files.cow.fi/tokens/CowSwap.json',
+      //   'https://tokens.coingecko.com/uniswap/all.json',
+      // ],
+      tradeType: TradeType.SWAP, // TradeType.SWAP, TradeType.LIMIT or TradeType.ADVANCED
+      sell: sell
+        ? sell
+        : {
+            // Sell token. Optionally add amount for sell orders
+            asset: '',
+            amount: '0',
+          },
+      enabledTradeTypes: [TradeType.SWAP, TradeType.LIMIT],
+      // env: 'dev',
+      theme: {
+        baseTheme: darkMode ? 'dark' : 'light',
+        primary: palette.primary.main,
+        background: palette.background.main,
+        paper: palette.background.paper,
+        text: palette.text.primary,
+        danger: palette.error.dark,
+        info: palette.info.main,
+        success: palette.success.main,
+        warning: palette.warning.main,
+        alert: palette.warning.main,
+      },
+      partnerFee: {
+        bps: 50,
+        recipient: '0x0B00b3227A5F3df3484f03990A87e02EbaD2F888',
```

Comment:
this is a smart contract... shouldn't you change this depending on the network? I can see is not deployed in Gnosis Chain... https://gnosisscan.io/address/0x0B00b3227A5F3df3484f03990A87e02EbaD2F888